### PR TITLE
Wrote resources folder tests

### PR DESCRIPTION
**Pull request by UnknownNPC**
Some tests for resources files number validation

### CODE REVIEW

File: src/test/java/org/mafagafogigante/dungeon/io/ResourcesFolderTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        } else {
+            Assert.fail("Unable to find resources folder");
+        }
+    }
+
+    @Test
+    public void testIsResourcesFolderContainsCorrectNumberOfJsonFiles() {
+
+        List<File> jsonFiles = findFilesByFileFilter(getEndWithFileFilter(JSON_FILE_EXTENSION));
+        Assert.assertEquals(10, jsonFiles.size());
```
Comment (mafagafogigante) [inline src/test/java/org/mafagafogigante/dungeon/io/ResourcesFolderTest.java]: 10 here is a nontrivial constant that we might need to change even if the current resource design does not change. It should be a private static member.

File: src/test/java/org/mafagafogigante/dungeon/io/ResourcesFolderTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    private static final String JSON_FILE_EXTENSION = ".json";
+    private static final String FONT_FILE_EXTENSION = ".ttf";
+    private static final String SPLASH_SCREEN_FILE_NAME = "splash.png";
+
+    private File resourcesDir;
+
+    @Before
+    public void initialize() {
+
```
Comment (mafagafogigante) [inline src/test/java/org/mafagafogigante/dungeon/io/ResourcesFolderTest.java]: I don't understand this and the other few blank lines at the beginning of the methods. I would remove them.

**Comment by mafagafogigante** [comment]
Looks done. I will manually review it and merge soon.

Thank you.

**Comment by UnknownNPC** [comment]
You're welcome. Also, thank you for quick response. 
If you have some issues from backlog, I can try to implement some.

**Comment by mafagafogigante** [comment]
There are several open issues I use to track ideas around here. Some of them are much easier and simpler to get right than others.

https://github.com/mafagafogigante/dungeon/issues/266 would be nice. But it has to change the version string everywhere or find a clever way to do it.

https://github.com/mafagafogigante/dungeon/issues/172 is what you started working on. There is also some more finely grained testing that must be done. The JSON files are proper JSON. The values are valid everywhere. Figuring out a good way to test these issues is important, as otherwise only during gameplay someone will notice that things aren't working as they should be.

**Comment by mafagafogigante** [comment]
These have been merged now. I manually adjusted the first commit to the address you provided for the second (unknownnpc@gmail.com), just so that Git history is clear.

**Pull request closed without merging**